CMakeLists.txt

```diff
@@ -43,6 +43,9 @@ target_link_libraries(console_calibration
   tool_point_calibration
 )
 
+add_executable(tcp_orientation_calibration  src/tcp_orientation_calibration.cpp)
+add_dependencies(tcp_orientation_calibration ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(tcp_orientation_calibration  ${catkin_LIBRARIES}  ${CERES_LIBRARIES}  tool_point_calibration)
```

Comment:
The naming of the node does not follow the same format as the `tool_orientation_calibration` node, could this be changed to be more consistent? This could also mean to change to the name of the `tool_orientation_calibration` node and even the package itself if that makes sense. @gavanderhoorn Any thoughts on this?